INSTALL.md

```diff
@@ -0,0 +1,38 @@
+#If you are in a hurry
+
+On Linux (Debian):
+
+```sh
+ $ [sudo] apt-get install libgrpc-dev
+```
+
+On MacOS:
+
+```sh
+ $ curl -fsSL https://goo.gl/getgrpc | bash -
```

Comment:
We should direct users to language specific packages if that's what they are actually interested in.
If you are in a Hurry and only want to install gRPC as a dependency, follow language specific instructions.
For the C core, this can be accomplished by doing the following.
Windows
Linux
Mac OSX..